TSOClient/FSOFacadeWorker/Program.cs

```diff
@@ -142,10 +137,10 @@ static void Main(string[] args)
         private static bool LoginSent;
         private static int Done;
 
-        public static void Login()
+        public static async Task LoginAsync()
         {
             api = new ApiClient(Config.Api_Url);
-            api.AdminLogin(Config.User, Config.Password, (result) =>
+            await api.AdminLoginAsync(Config.User, Config.Password, async (result) =>
```

Comment:
If you discard the Task with `_ = await api.AdminLoginAsync...`, this method doesn't need to be async.